src/migrations/1741025559300_scene-admin.ts

```diff
@@ -1,24 +0,0 @@
-import { MigrationBuilder, ColumnDefinitions } from 'node-pg-migrate'
-
-export const SceneAdminColumns: ColumnDefinitions = {
-  id: { type: 'uuid', primaryKey: true },
-  place_id: { type: 'text', notNull: true },
-  admin: { type: 'text', notNull: true },
-  added_by: { type: 'text', notNull: true },
-  created_at: { type: 'bigint', notNull: true },
-  active: { type: 'boolean', notNull: true, default: true }
```

Comment:
Do we need to keep the rows of the tokens that are not active anymore? What if we remove them from the DB?

Reply:
We can do it maybe after some time. I'm thinking in the notifications, maybe we'll need something from those rows